packages/shared/components/Search/SearchPanel.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { Text, Flex } from 'design';
+import { StyledPanel } from 'design/DataTable';
+import { PageIndicatorText } from 'design/DataTable/Pager/Pager';
+import InputSearch from 'design/DataTable/InputSearch';
+import { AgentFilter } from 'teleport/services/agents';
+import Toggle from 'teleport/components/Toggle';
+import Tooltip from 'teleport/components/ServersideSearchPanel/Tooltip';
+
+import { PredicateDoc } from './PredicateDoc';
+
+export function SearchPanel({
```

Comment:
This component seems to get obliterated if you make the window narrow enough, but I suppose we can fix that at some later time.

![narrow search](https://user-images.githubusercontent.com/27113/203346019-106291e1-4167-47b4-9579-8c11797549a3.png)

Reply:
While I do agree it's not optimal, it isn't quite as bad on my end. I wonder if something was up with the styling in your build? I can't replicate these style errors you've had exactly. 

<img width="495" alt="Screenshot 2022-11-23 at 3 37 14 PM" src="https://user-images.githubusercontent.com/5201977/203485585-4e181129-df62-4ebd-9e28-28953029b9f1.png">

Reply:
You're right, something must have been wrong with my setup. I cannot reproduce this issue and the issue with the address now.